[PASMS-57] Refactored the Adopter component (sign in / sign up forms).

GeorgeBeshay opened the pull request:
# Introduction
Modified the CSS of the Sign Up / Sign In forms of the Adopter component, to match and be consistent with the previous forms already developed.

# Details
Modified the CSS file, in which I've removed the unnecessary styling (either was duplicated, not used or overwritten). I've also used global styling instead, that's why you will find a set of styling added to the global CSS file to style the sign in / sign up forms using the same CSS. The global CSS classes made have a **_"glb"_** prefix, which stands for "global".

Q: "Since you have already used global CSS styling, why didn't you remove them from the component CSS file ??"
A: Due to the presence of subpages in that component that still use this CSS.

# Attachments
![image](https://github.com/GeorgeBeshay/db-project/assets/90455303/237095bb-1487-4120-93d5-3fcfd272a026)
![image](https://github.com/GeorgeBeshay/db-project/assets/90455303/d1021635-d2b4-4041-895b-e7c0991c88b8)

GeorgeBeshay committed:
Refactored the Adopter component sign in / sign up forms.

bangkook reviewed (changes requested):
Good job, George. I only have some comments:
- Why didn't you use a date picker for the birthdate field? It's not only because it would be more user-friendly, but also it might not match the date format used in the backend which may cause a lot of errors until you figure it out correctly. It is better if you can enforce that format somehow.
- The boolean values for the gender need to be adjusted to correspond to (Male\Female) values. or just give any hint regarding what boolean value corresponds to what gender.
- I had a problem when I tried to sign in, saying that no records found for the given email although there is. So tell me if you faced the same problem or if this branch is not intended for such issues.

GeorgeBeshay committed:
Date and Gender fields modified.

GeorgeBeshay commented:
> Good job, George. I only have some comments:
> 
> * Why didn't you use a date picker for the birthdate field? It's not only because it would be more user-friendly, but also it might not match the date format used in the backend which may cause a lot of errors until you figure it out correctly. It is better if you can enforce that format somehow.
> * The boolean values for the gender need to be adjusted to correspond to (Male\Female) values. or just give any hint regarding what boolean value corresponds to what gender.
> * I had a problem when I tried to sign in, saying that no records found for the given email although there is. So tell me if you faced the same problem or if this branch is not intended for such issues.

---

## Requested Changes Implemented

@bangkook 
Considering the date and gender fields, I didn't notice them TBH, I was focused on refactoring the overall form display and not each field solely. However, you are totally right, so I've just modified the date field to use a date picker instead of the text-input, and changed the gender field to be a select menu. Please, find attached screenshots for the newest version of the form down below.

About your last note, I'm not sure what is the reason behind that, but I've tried to sign up / sign in and it works fine. I suspect there may be some inconsistency in the DB schema on our machines.

---

## Attachments
![image](https://github.com/GeorgeBeshay/db-project/assets/90455303/9ea04558-8b3b-496d-80f6-a66bf25eefe4)
![image](https://github.com/GeorgeBeshay/db-project/assets/90455303/89646e30-b299-4762-89ef-13ab85d61bc3)

---
